SteamBundleSitesExtension.js

```diff
@@ -830,8 +866,8 @@ const bundleSitesBox = () => {
 };
 const siteCache = {
     bundlestars: {
-        doms: [document],
-    },
+        doms: [document]
```

Comment:
Trailing comma needed

Reply:
_(:зゝ∠)_, why needed
(I'm not really good at it)

Reply:
That's the coding style I am using when coding this script, so good to be consistent throughout the script
You can install eslint on your editor and apply AirBnB coding style, it will warn you the changes should be made

Reply:
OK, I'll setup them and check my code